Task6.1/Catalog.cs

```diff
@@ -0,0 +1,97 @@
+﻿namespace BookCatalogTask
+{

```

Comment:
I think that this wrapper class is not necessary, at least now))

Reply:
Fixed :)